01_Core/week03/submissions/exercise 3/index.html

```diff
@@ -0,0 +1,74 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>CSS Exercise 3</title>
+  <meta name="description" content="ITA CSS Layout Exercise">
+  <meta name="author" content="Mark Kroh">
+
+  <style>
+  	.container  {
+		  width: 275px;
+		  height: 275px;
+		  position: relative;
+	  }
+
+	  .interior	{
+		  width: 100px;
+		  height: 100px;
+		  margin: 10px 10px;
+	  }
+
+	  .div1	{
+		  background-color: blue;
+		  display: inline-block;
+		
+	  }
+
+	  .div2	{
+		  background-color: blue;
+		  display: inline-block;
+		  
+	  }
+
+	  .div3	{
+		  background-color: blue;
+		  display: inline-block;
+		  
+	  }
+
+	  .div4	{
+		  background-color: blue;
+		  display: inline-block;
+	  }
+
+	  .div5	{
+		  background-color: red;
+		  position: absolute;
+		  top: 62px;
+		  left: 62px;
+		  transform: rotate(45deg);
+	  }
+
+	  .div5::after	{
+		  content: "\22C6";
+		  position: absolute;
+		  display: flex;
+		  height: 80px;
```

Comment:
Height and width aren't actually doing anything to change the size of your unicode character here - `font-size` would allow you to change the size of the icon.
For positioning it in the center, if you want to use flexbox, I believe you would actually need to put `display: flex;` on the parent container, which in this case would be `.div5` instead of the `::after`  element. Or you could just use absolute positioning on the `::after` element without flexbox to get it where you want it